core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/SynapseTests.scala

```diff
@@ -109,17 +108,17 @@ class SynapseTests extends TestBase {
     .filterNot(_.getAbsolutePath.contains("CyberML"))
     .filterNot(_.getAbsolutePath.contains("VowpalWabbitOverview"))
     .filterNot(_.getAbsolutePath.contains("IsolationForest"))
-    .filterNot(_.getAbsolutePath.contains("ExplanationDashboard"))
     .filterNot(_.getAbsolutePath.contains("DeepLearning"))
     .filterNot(_.getAbsolutePath.contains("InterpretabilitySnowLeopardDetection"))
     .sortBy(_.getAbsolutePath)
 
-  selectedPythonFiles.foreach(println)
-  assert(selectedPythonFiles.length > 1)
```

Comment:
This is necessary to avoid bugs where we dont point at the correct file

Reply:
Both of these are already covered in lines 117-119